### ISSUE TITLE
BP1658CJ max power limit

### ISSUE THREAD

**Issue by kogeki** [openshwprojects/OpenBK7231T_App#686]
Hello,

I am reading the documentation for ESPHome and I see that the [BP1658CJ](https://esphome.io/components/output/bp1658cj.html) driver has the option to increase the max power draw. How can I set this setting with OpenBK? Thanks!

**Comment by openshwprojects** [comment]
Hello, it seems that the command has to be yet added. I can try to do that today, can you help with testing?

**Comment by kogeki** [comment]
Yes I can.

**Comment by kogeki** [comment]
Not to pressure you (I'm sure you have other things to worry about), but is there any chance you can look at this? Thanks!

**Comment by openshwprojects** [comment]
I don't have this device at hand, all I can do is add command and let you test, but I will need multiple tests...
I added BP1658CJ_Current that takes two arguments and need light change after that to apply (to refresh),
try BP1658CJ_Current 2 2 or something like that.
I am not sure if it will work, there is also option that I might to do one more change. Please check and let me know how it behaves.

**Comment by kogeki** [comment]
I appreciate the response. After trying it out with various values, I have noticed that while it does make the light brighter (sometimes), it also makes the LED Temp slider behave very erratically. 
![image](https://user-images.githubusercontent.com/17801223/222922373-21b6fa61-c146-4292-8083-4bc8f1701619.png)
^this makes the light turn off (`BP1658CJ_Current 2 2`)
![image](https://user-images.githubusercontent.com/17801223/222922383-2262debd-2126-4f07-97a8-c443409711c4.png)
^this is really dim warm white (`BP1658CJ_Current 2 2`)
Using other values just makes them have the same behavior but brighter or dimmer.

Any other things I can try? Thanks!

**Comment by openshwprojects** [comment]
We need to try things but I don't have this device. Do you know how to download build from a pull request?
Try that: https://github.com/openshwprojects/OpenBK7231T_App/pull/708
here, at the bottom:
https://github.com/openshwprojects/OpenBK7231T_App/actions/runs/4332532167
![image](https://user-images.githubusercontent.com/85486843/222925557-890444a1-3e59-49d3-ae32-4f12a38881f8.png)

download this build and try again

**Comment by kogeki** [comment]
This version works perfectly. Thanks a lot! I am wondering though, what does each value mean? I am running with `15 15` and it seems to be equal to the real light.

**Comment by openshwprojects** [comment]
wow, that's great, so I managed to get it without device on the second try.

	//cmddetail:{"name":"BP1658CJ_Current","args":"[MaxCurrentRGB][MaxCurrentCW]",

i will merge this into main tree tomorrow

**Comment by kogeki** [comment]
Thanks so much! Do you happen to have a crypto currency donation address? I would be glad to send some money. Thanks for this great project!

**Comment by openshwprojects** [comment]
No, only Paypal, do you prefer another donation method? If so, what kind of method and why? I always wanted to get into that crypto thing, but never had time to look into that well.
https://www.paypal.com/paypalme/openshwprojects

**Comment by kogeki** [comment]
I am underage and my PayPal account is limited. I can't send money. Bitcoin would be the best option, if you can.
